oscarapi/serializers/checkout.py

```diff
@@ -187,6 +187,8 @@ class OrderSerializer(OscarHyperlinkedModelSerializer):
     shipping_address = InlineShippingAddressSerializer(many=False, required=False)
     billing_address = InlineBillingAddressSerializer(many=False, required=False)
 
+    email = serializers.CharField()
```

Comment:
There is a dedicated email field in DRF: https://www.django-rest-framework.org/api-guide/fields/#emailfield

Reply:
https://github.com/django-oscar/django-oscar/blob/master/src/oscar/apps/order/abstract_models.py#L335
Property email on the Order model

I made the email serializer field readonly because you can't write to a property.